src/app/index.ts

```diff
@@ -19,21 +18,22 @@ import { upgradeAdapter } from "./adapter";
 require("./vendor.scss");
 require("./index.scss");
 
-import config from "./config.provider";
-import signin from "./signin";
+upgradeAdapter.upgradeNg1Provider("hotkeys"); // needed for pushToTalk
 
-// Components
-import footerComponent from "./footer";
-import feedComponent from "./feed";
-import chatComponent from "./chat";
-import roomComponent from "./room";
+/* Register providers for browser, this is mandatory. */
```

Comment:
I'm a little bit confused by this comment. It claims that something is mandatory... but it's commented.